pandas/core/generic.py

```diff
@@ -4099,7 +4099,6 @@ def add_suffix(self: FrameOrSeries, suffix: str) -> FrameOrSeries:
 
     def sort_values(
         self,
-        by=None,
```

Comment:
DataFrame.sort_values is not typed. This signature will not now be compatible?

Reply:
mypy now gives (when DataFrame.sort_values is typed)
pandas\core\frame.py:4867: error: Signature of "sort_values" incompatible with supertype "NDFrame"
instead of 
pandas\core\series.py:2776: error: Signature of "sort_values" incompatible with supertype "NDFrame"

so deleting this is not solving the issue.

Reply:
Note there is a dedicated issue #30451 for sort_values; would be OK with tackling that on its own as I think will be tricky

Reply:
@topper-123 can you restored this deleted annotation? IIUC from @simonjayhawkins above doesn't need to be removed

Otherwise lgtm happy to get this merged